src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/service/CapacityCalculator.kt

```diff
@@ -1,30 +1,26 @@
 package uk.gov.justice.digital.hmpps.hmppsworkload.service
 
-import org.springframework.stereotype.Service
```

Comment:
This doesn't have any Spring managed dependencies, and the two functions in it are independent of each other and never used together. This refactor allows each to be imported as required.